lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -2543,6 +2546,29 @@ createAcceleratorReceive(SILBuilder &B, SILLocation loc, SILType valueTy,
   return B.createBuiltin(loc, name, valueTy, subMap, operands);
 }
 
+static SILValue createAcceleratorReceiveViaGraphOp(
```

Comment:
The "ViaGraphOp" suffix will eventually become redundant.  would it make more sense to add "Legacy" to the one you plan to remove in the future?

Reply:
Since i'll be touching the code again when we delete the old code, i'll defer to the renaming at that time.